hive/README.md

```diff
@@ -0,0 +1,77 @@
+# Agent Check: hive
+
+## Overview
+
+This check monitors two parts of [hive][1]: Hive Metastore and HiveServer2.
+
+## Setup
+
+### Installation
+
+The hive check is included in the [Datadog Agent][2] package.
```

Comment:
Some links are wrong

Reply:
The wrong links are those pointing at files on the master branch. Since Hive is not merged yet, I guess they should be wrong until the merge

Reply:
Not what I mean 🙂 

For example 2 is https://github.com/DataDog/integrations-core/blob/master/hive/datadog_checks/hive/data/conf.yaml.example